spec/join.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function (array) {
+  for (i = 0; i < array.length; i++) {
+
```

Comment:
this function is not passing the test ...***hint*** probably should use ```push()``` somewhere in here